editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/BeanPropertiesFilter.java

```diff
@@ -30,7 +30,10 @@ public boolean select(Object toTest) {
 
 	protected AbstractCamelModelElement getSelectedObject(Object toTest) {
 		if (toTest instanceof CamelBean) {
-			return (CamelBean)toTest;
+			CamelBean testBean = (CamelBean) toTest;
+			if (!testBean.getClassName().equals("org.fusesource.camel.component.sap.SapConnectionConfiguration")) { //$NON-NLS-1$
```

Comment:
better to invert equals even more that i'm not sure that the CamelBean will always return a non-null value as if a bean is defined but the classname is notprovided yet